rpc/cc/arduino/cli/commands/v1/common.proto

```diff
@@ -91,6 +91,9 @@ message PlatformSummary {
   string installed_version = 3;
   // The latest available version of the platform, or empty if none available
   string latest_version = 4;
+  // The latest available version of the platform containing all compatible
+  // dependencies, or empty if none available
+  string latest_compatible_version = 5;
```

Comment:
I'm not sure about this field, I'm for removing it (the latest non-compatible version, if really needed, could be determined from the `releases` map when the request is done with `all_version==true` set).

Reply:
>I'm not sure about this field, I'm for removing it (the latest non-compatible version, if really needed, could be determined from the releases map when the request is done with all_version==true set).

@kittaakos WDYT? :point_up:

Reply:
No problem. Please remove it if the info can be inferred.

The `latest_version` can be inferred too 🙃